### PR TITLE
update for alpha4

### PR DESCRIPTION
**Pull request by talevy**
I've been playing around with this plugin and wanted to use it against master.

I don't think this should be merged until alpha4 is released, and I can remove the dependency on the SNAPSHOT.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,10 +32,7 @@ esplugin {
 
 // In this section you declare the dependencies for your production and test code
 dependencies {
-  compile 'org.elasticsearch:elasticsearch:5.0.0-alpha3'
```
Comment (talevy) [inline build.gradle:35]: I was told that these are unnecessary since this is included by the build-tools in the buildscript

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "org.elasticsearch.gradle:build-tools:5.0.0-alpha3"
+    classpath "org.elasticsearch.gradle:build-tools:5.0.0-alpha4-SNAPSHOT"
```
Comment (talevy) [inline build.gradle:9]: needs to change to alpha4 next week, when it is released.

**Comment by spinscale** [comment]
yeah, I intend to update this with stable releases only.. so once alpha4 is out, I will do the update

File: src/test/resources/rest-api-spec/test/ingest_opennlp/10_basic.yaml [hunk 3]
```diff
@@ -8,21 +8,5 @@
...
-    - match:  { nodes.$master.ingest.processors.8.type: lowercase }
-    - match:  { nodes.$master.ingest.processors.9.type: opennlp }
-    - match:  { nodes.$master.ingest.processors.10.type: remove }
-    - match:  { nodes.$master.ingest.processors.11.type: rename }
-    - match:  { nodes.$master.ingest.processors.12.type: set }
-    - match:  { nodes.$master.ingest.processors.13.type: sort }
-    - match:  { nodes.$master.ingest.processors.14.type: split }
-    - match:  { nodes.$master.ingest.processors.15.type: trim }
-    - match:  { nodes.$master.ingest.processors.16.type: uppercase }
+    - match:  { nodes.$master.ingest.processors.0.type: opennlp }
```
Comment (spinscale) [inline src/test/resources/rest-api-spec/test/ingest_opennlp/10_basic.yaml:11]: did we stop exposing the built-in processors?

Comment (talevy) [inline src/test/resources/rest-api-spec/test/ingest_opennlp/10_basic.yaml:11]: I think this is a result of all the processors moving into a module, `ingest-common`. I guess modules are not included during the integration test distribution bundling

**Pull request closed without merging**